teardown.js

```diff
@@ -0,0 +1,60 @@
+var fs = require("fs");
+var spawn = require('child_process').spawn;
+var wasInArmMode = false;
+
+readUUIDFromConfigFile();
+var errorHasOccurred = false;
+var connectionData = "";
+var wasInArmMode = false;
+
+function readUUIDFromConfigFile() {
+  fs.readFile('resources/config.properties', (err, data) => {
+    if (err) throw err;
+    var fileData = `${data}`;
+    var firstLineIndex = fileData.indexOf("\n");
+    if(firstLineIndex > 0) {
+      var uuidrg = JSON.parse(fileData.substring(1, firstLineIndex + 1)).uuid;
+      executeAzureCLI(['config', 'list'], function() {
+        executeAzureCLI(['config', 'mode', 'arm'], function() {
+          var resourceGroup = "rg" + uuidrg.substring(0,19);
+          executeAzureCLI(['group', 'delete', '-q', resourceGroup], function() {
+              if(!wasInArmMode) {
```

Comment:
Should probably also clean up the config file in here. Otherwise they're left with a connection string that doesn't work and the random uuidrg.